packages/fractor/src/Helper/ArrayUtility.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\Fractor\Helper;
+
+class ArrayUtility
```

Comment:
Even it is copied from TYPO3, we should add some Tests for it. You can basically copy the tests from TYPO3. I would also drop all parameters which we do not use at the moment. Adds a lot of unnecessary complexity.